fe/src/layouts/default.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+  <div>
+    <slot />
+  </div>
+</template>
+
+<script setup lang="ts"></script>
+
+<style scoped></style>
```

Comment:
Since we're using tailwind, `<style scoped />` will no longer be utilized. Please remove.